commands/user_management.js

```diff
@@ -0,0 +1,55 @@
+const settings = require('../settings/settings.json')
+const {
```

Comment:
Can these not be encapsulated into one export? It seems weird to my object-oriented mind to consistently have multiple exports but I'm not sure if this is idiomatic for JS.

Reply:
I don’t know how idiomatic it is... I did copy it from other examples I found. I think JS is perfectly happy to treat functions as first class, just as much as objects. 

Have you read [Steve Yegge’s famous post about OO and Java](http://steve-yegge.blogspot.com/2006/03/execution-in-kingdom-of-nouns.html)?
> Verbs in Javaland are responsible for all the work, but as they are held in contempt by all, no Verb is ever permitted to wander about freely. If a Verb is to be seen in public at all, it must be escorted at all times by a Noun.